keyboards/honeycomb/honeycomb.h

```diff
@@ -0,0 +1,40 @@
+#ifndef honeycomb_H
+#define honeycomb_H
+
+#include "quantum.h"
+#include "matrix.h"
+#include "backlight.h"
+#include <stddef.h>
+
+#define red_led_off   PORTF |= (1<<6)
```

Comment:
Two things here.

Could you make sure that they are defined so that it appears as a function? 

The second is, please use the GPIO controls.

```suggestion
#define red_led_off()   writePinHigh(F6)
```

Reply:
```suggestion
#define RED_LED_OFF()   writePinHigh(F6)
```
Please use `CONSTANT_CASE` for preprocessor constant/macro names.